shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -360,13 +362,13 @@ instance NoUnexpectedThunks (UTxOState hashAlgo dsignAlgo vrfAlgo)
 -- | New Epoch state and environment
 data NewEpochState hashAlgo dsignAlgo vrfAlgo =
   NewEpochState {
-    nesEL    :: Epoch
-  , nesEta0  :: Nonce
-  , nesBprev :: BlocksMade hashAlgo dsignAlgo
-  , nesBcur  :: BlocksMade hashAlgo dsignAlgo
-  , nesEs    :: EpochState hashAlgo dsignAlgo vrfAlgo
-  , nesRu    :: Maybe (RewardUpdate hashAlgo dsignAlgo)
-  , nesPd    :: PoolDistr hashAlgo dsignAlgo vrfAlgo
+    nesEL     :: Epoch                                   -- ^ Epoch index
+  , nesEta0   :: Nonce                                   -- ^ Epoch nonce
+  , nesBprev  :: BlocksMade hashAlgo dsignAlgo           -- ^ Blocks made before current epoch
+  , nesBcur   :: BlocksMade hashAlgo dsignAlgo           -- ^ Blocks made in current epoch
+  , nesEs     :: EpochState hashAlgo dsignAlgo vrfAlgo   -- ^ Epoch state before current
+  , nesRu     :: Maybe (RewardUpdate hashAlgo dsignAlgo) -- ^ Possible reward update
+  , nesPd     :: PoolDistr hashAlgo dsignAlgo vrfAlgo
```

Comment:
stake pool stake distribution

Reply:
and below is the overlay schedule for PBFT vs Praos

Reply:
Great thanks!